diff --git a/dist/index.js b/dist/index.js
index 51c24d31e8121c2b1d9d89c0acc86f63c9fa65a4..bc1817ceb59bd665f462b06dc3cf17463eb17400 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -11,7 +11,7 @@ import node_path, { isAbsolute as external_node_path_isAbsolute, sep as external
 import node_assert from "node:assert";
 import node_v8 from "node:v8";
 import node_fs_promises from "node:fs/promises";
-import { transform } from "typia/lib/transform.js";
+import { transform } from "typia/lib/transform";
 import { createHash } from "node:crypto";
 import node_os from "node:os";
 var __webpack_modules__ = {
